bskyweb/cmd/bskyweb/server.go

```diff
@@ -336,12 +336,20 @@ func (srv *Server) WebPost(c echo.Context) error {
 	postView := tpv.Thread.FeedDefs_ThreadViewPost.Post
 	data["postView"] = postView
 	data["requestURI"] = fmt.Sprintf("https://%s%s", req.Host, req.URL.Path)
-	if postView.Embed != nil && postView.Embed.EmbedImages_View != nil {
-		var thumbUrls []string
-		for i := range postView.Embed.EmbedImages_View.Images {
-			thumbUrls = append(thumbUrls, postView.Embed.EmbedImages_View.Images[i].Thumb)
+	if postView.Embed != nil {
+		if postView.Embed.EmbedImages_View != nil {
+			var thumbUrls []string
+			for i := range postView.Embed.EmbedImages_View.Images {
```

Comment:
you kept it the way it was before, so this is fine, but the code would be more compact if this iterator was `for _, img := range postView.Embed.EmbedImages_View.Images`, so the long variable names don't need to be indexed below

Reply:
Yes but that would require me to actually know Go which I decidedly don't, I just looked at the code and sorta inferred how the language works 😁

Reply:
all the more impressed!